csharp/src/Microsoft.ML.OnnxRuntime/SessionOptions.cs

```diff
@@ -7,10 +7,17 @@
 
 namespace Microsoft.ML.OnnxRuntime
 {
+    public enum GraphOptimizationLevel
+    {
+        ORT_DISABLE_ALL,
+        ORT_ENABLE_BASIC,
+        ORT_ENABLE_ALL
```

Comment:
Set explicit values so adding something in the middle doesn't screw things up.

Reply:
Sure.